legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/jdbc/JdbcHelper.java

```diff
@@ -80,12 +83,24 @@ public Connection connection()
         return connection;
     }
 
+    protected RelationalTransactionManager intializeTransactionManager(Connection connection)
```

Comment:
Great idea, let's just not have try catch here as the methods which call this method already catch SqlException.

protected RelationalTransactionManager intializeTransactionManager(Connection connection) throws SqlException